index.css

```diff
@@ -23,23 +23,83 @@ ul li {
 #nav-container {
   display: flex;
   align-items: center;
+  position: relative;
+}
+
+#nav_list_container {
+  display: none;
+  flex-direction: column;
+  text-align: center;
+  padding-right: 5em;
+  gap: 3%;
+  width: 100%;
 }
 
-.nav_list_container {
+.desktop_nav_list_container {
   display: none;
 }
 
-.shape1 {
+a {
+  text-decoration: none;
+}
+
+#close-btn {
+  display: none;
+  align-self: flex-end;
+  margin-bottom: 7em;
+  padding-right: 1em;
+  position: relative;
+  right: -4em;
+  top: 0;
+}
+
```

Comment:
- [ ] **Good design so far :clap: 👏** However, it would be better to the colors of your mobile menu to that of the Figma design 🙏 

**Figma design** | **Your design**
| --- | --- |
![image](https://user-images.githubusercontent.com/102757126/223248874-96eabd0a-c7e9-42f4-9055-b3e615454458.png) | ![image](https://user-images.githubusercontent.com/102757126/223249247-c99fed9b-6a4b-45db-8158-0edaeee91f3d.png)